akka-docs/src/main/paradox/cluster-dc.md

```diff
@@ -112,11 +112,11 @@ Two different failure detectors can be configured for these two purposes:
 * `akka.cluster.failure-detector` for failure detection within own data center
 * `akka.cluster.multi-data-center.failure-detector` for failure detection across different data centers
 
-When @ref[subscribing to cluster events](cluster-usage.md#cluster-subscriber) the `UnreachableMember` and
-`ReachableMember` events are for observations within the own data center. The same data center as where the
+When @ref[subscribing to cluster events](cluster-usage.md#cluster-subscriber) the @scala[@scaladoc[`UnreachableMember`](akka.cluster.ClusterEvent$)]@java[`UnreachableMember`] and
+@scala[@scaladoc[`ReachableMember`](akka.cluster.ClusterEvent$)]@java[@javadoc[`ReachableMember`](akka.cluster.ClusterEvent$)] events are for observations within the own data center. The same data center as where the
```

Comment:
### scala
scaladoc is linked to:
https://doc.akka.io/api/akka/2.5/akka/cluster/ClusterEvent$.html

As the URL for `UnreachableMember` includes the return type, which is messy.
https://doc.akka.io/api/akka/2.5/akka/cluster/ClusterEvent$.html#UnreachableMemberextendsClusterEvent.ReachabilityEventwithProductwithSerializable

### java
javadoc link is not added as there is no way to get it work. 
The correct URL is:
https://doc.akka.io/japi/akka/2.5/akka/cluster/ClusterEvent.UnreachableMember.html

but `@javadoc[...](akka.cluster.ClusterEvent.UnreachableMember)`  will be linked to 
https://doc.akka.io/japi/akka/2.5/akka/cluster/ClusterEvent/UnreachableMember.html (`/` after ClusterEvent) 
which doesn't exist.

Reply:
And the class name from Java's point of view does not work either? I think that'd be
`akka.cluster.ClusterEvent$.UnreachableMember`